LoRaEngine/test/LoRaWan.IntegrationTest/LoraArduinoSerial.cs

```diff
@@ -1,12 +1,15 @@
-﻿/*
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Please revert changes to pascal case in method names. The reason we keep this is that we ported if from a c library, and to make future work easier they should match as much as possible the original.
To avoid StyleCop complaining about it add the required elements in this file.

Reply:
Done. Disabled the following rules in LoraArduinoSerial.cs:
#pragma warning disable SA1300 // Elements should begin with an uppercase letter
#pragma warning disable SA1313 // Parameters should begin with a lowercase letter